src/ex4/server/routes/api.js

```diff
@@ -5,8 +5,18 @@ import ItemManager from '../services/item_manager.js'
 const router = express.Router()
 const itemManager = new ItemManager()
 
-router.get('/', itemManager.getItems)
-router.post('/', itemManager.postItem)
-router.delete('/:id', itemManager.deleteItem)
+router.get('/', (req, res) => {
```

Comment:
I actually preferred the solution you had before, without specifically mentioning (req, res), I think it's cleaner